package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@mondaycom/apps-sdk",
-  "version": "2.1.2",
+  "version": "2.1.3",
```

new feature => bump minor
bug fix => bump patch
```suggestion
  "version": "2.2.0",
```